pages/learn/develop/dockerfile.md

```diff
@@ -128,34 +79,25 @@ RUN npm install
 COPY src/ /usr/src/app
 CMD ["node", "/usr/src/app/main.js"]
 ```
-This template will build and deploy a node.js project for any of the devices supported by resin.io, regardless of whether the device architecture is [ARM][ARM-link] or [x86][x86-link].
-In this example you can see the build variable `%%RESIN_MACHINE_NAME%%` this will be replaced by the machine name (i.e.: raspberrypi) at build time. See below for a list of machine names.
+This template will build and deploy a Node.js project for any of the devices supported by resin.io, regardless of whether the device architecture is [ARM][ARM-link] or [x86][x86-link].
+In this example, you can see the build variable `%%RESIN_MACHINE_NAME%%`. This will be replaced by the machine name (i.e.: `raspberry-pi`) at build time. See below for a list of machine names.
 
- The machine name is inferred from what "device type" application you are pushing to. So if the resin remote you are pushing to is associated to an Intel Edison application, the machine name will be `edison` and an `i386` architecture base image will be built.
+ The machine name is inferred from the device type of the application you are pushing to. So if you have an Intel Edison application, the machine name will be `intel-edison` and an `i386` architecture base image will be built.
 
-__Warning:__ One caveat to this is that you need to ensure that your dependencies and node.js modules are also multi-architecture, otherwise you will have a bad time.
+__Note:__ You need to ensure that your dependencies and Node.js modules are also multi-architecture, otherwise you will have a bad time.
 
 Currently our builder supports the following build variables.
 
 | Variable Name        | Description          |
 | ------------- |:-------------:|
-| RESIN_ARCH    | This is the CPU architecture of the fleet. This is defined when you select a device type in the dashboard while creating an application.|
-| RESIN_MACHINE_NAME    | This is the name of the yocto machine this board is base on. It is the name that you will see in most of the resin Docker Hubbase images.  This name helps us identify a specific [BSP](https://en.wikipedia.org/wiki/Board_support_package). |   
-
+| RESIN_MACHINE_NAME    | This is the name of the yocto machine this board is base on. It is the name that you will see in most of the resin.io [Docker base images][base-images].  This name helps us identify a specific [BSP](https://en.wikipedia.org/wiki/Board_support_package). | 
+| RESIN_ARCH    | This is the CPU architecture associated with the device type.|
```

Comment:
I am trying to come up with a better description for RESIN_ARCH, but I am struggling a bit. CPU architecture is a bit misleading here, especially because RPi3 has ARMv8 CPU, but our current host OS and container images for RPi3 are 32-bit. So something along the lines of "Instruction set architecture of the base image". Ideas?